lib/commons/dom/get-overflow-hidden-ancestors.js

```diff
@@ -0,0 +1,28 @@
+import memoize from '../../core/utils/memoize';
+
+/**
+ * Get all ancestor nodes (including the passed in node) that have overflow:hidden
+ * @method getOverflowHiddenAncestors
+ * @memberof axe.commons.dom
+ * @param {VirtualNode} vNode
+ * @returns {VirtualNode[]}
+ */
+const getOverflowHiddenAncestors = memoize(
+  function getOverflowHiddenAncestorsMemoized(vNode) {
+    const ancestors = [];
+
+    if (!vNode) {
+      return ancestors;
+    }
+
+    const overflow = vNode.getComputedStylePropertyValue('overflow');
+
+    if (overflow === 'hidden') {
```

Comment:
Should have a ticket for overflow-x: hidden overflow-y: visible.

Reply:
https://github.com/dequelabs/axe-core/issues/3818